solidity/random-beacon/contracts/libraries/Heartbeat.sol

```diff
@@ -66,14 +66,13 @@ library Heartbeat {
         SortitionPool sortitionPool,
         FailureClaim calldata claim,
         Groups.Group storage group,
-        uint256 nonce
+        uint256 nonce,
+        uint32[] calldata members
     ) external view returns (uint32[] memory failedMembers) {
-        uint256 groupSize = group.members.length;
-
         // Validate failed members indices. Maximum indices count is equal to
```

Comment:
Aren't we missing list vs hash validation here?

Reply:
It is validated earlier, in `RandomBeacon.notifyFailedHeartbeat`: 

https://github.com/keep-network/keep-core/blob/d830ff39a27cbc0657f780ba16f5356aaf7964c7/solidity/random-beacon/contracts/RandomBeacon.sol#L905-L908

Reply:
We validate it upstream just like Piotr mentioned

Reply:
Great! Can we add a comment inline here to clarify?

Reply:
fa1bd7f